packages/pcre/pcre.7.1.6/files/pcre.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
My understanding of `etc` is that it's switch-global and not package specific.  In that case, saving `setup.data` in `/etc` would get overwritten by another installation